### PR TITLE
feat: add whats new section (WPB-3777)

### PR DESCRIPTION
**Pull request by saleniuk**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-3777" title="WPB-3777" target="_blank">StoryWPB-3777</a>  What's new section in the navigation tab
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [x] contains a reference JIRA issue number like `SQPIT-764`
    - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

As wire, we want to improve our communication with the customer and highlight our new features by having a “What’s new“ section in the app.

### Solutions

Created new screen and item on drawer menu called "what's new". Added two static url destinations for "welcome to wire's new android app" and "all android release notes" with links to proper articles, and used RSS parser library to fetch recent wire android release notes from medium. Also, according to the acceptance criteria, "give feedback" and "report bug" items are moved to the ""other" settings section.

#### How to Test

Open the drawer menu and navigate to "what's new" section.

### Attachments (Optional)

https://github.com/wireapp/wire-android-reloaded/assets/30429749/d9d69f61-01db-4fb7-8454-8b899bd46ee7

----
#### PR Post Submission Checklist for internal contributors (Optional)

- [x] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

- [x] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [676](https://10.10.124.134/job/AR-build-pipeline/676/) **failed**.

**Comment by AndroidBob** [comment]
Build [678](https://10.10.124.134/job/AR-build-pipeline/678/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.5.0(7522159)(7522159)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.5.0(7522159)(7522159)-staging-compat.apk)

File: app/src/main/kotlin/com/wire/android/ui/home/whatsnew/WhatsNewViewModel.kt [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+) : ViewModel() {
+    private val rssParser = RssParser()
+    private val publishDateFormat = SimpleDateFormat("EEE, dd MMM yyyy HH:mm:ss", Locale.ENGLISH)
+
+    var state by mutableStateOf(WhatsNewState())
+        private set
+
+    init {
+        viewModelScope.launch {
+            rssParser.getRssChannel(context.getString(R.string.url_rss_android_release_notes)).let {
```
Comment (MohamadJaara) [inline app/src/main/kotlin/com/wire/android/ui/home/whatsnew/WhatsNewViewModel.kt]: Suggestion: maybe moving the link to the default.json is more appropriate

Comment (saleniuk) [reply app/src/main/kotlin/com/wire/android/ui/home/whatsnew/WhatsNewViewModel.kt]: Right, moved release notes rss url to default.json

**Comment by AndroidBob** [comment]
Build [685](https://10.10.124.134/job/AR-build-pipeline/685/) **failed**.

**Comment by AndroidBob** [comment]
Build [686](https://10.10.124.134/job/AR-build-pipeline/686/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.5.0(7547871)(7547871)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.5.0(7547871)(7547871)-staging-compat.apk)

**Review by alexandreferris — APPROVED** [review]

**Review by MohamadJaara — APPROVED** [review]

**Pull request merged**